internal/runners/install/install.go

```diff
@@ -114,6 +114,7 @@ func (i *Install) Run(params Params) (rerr error) {
 	var oldCommit *bpModel.Commit
 	var reqs requirements
 	var ts time.Time
+	var override bool
```

Comment:
I think for the install action we always want to override.

Reply:
Ah okay, so anytime we make a change we want to update the timestamp? I was under the impression that we want to preserve the timestamp in the solve node and I was only bumping it when the user explicitly passed the `--ts` flag.

Reply:
We only want to preserve it when we retrieve the commit. Once we're acting on the buildscript (ie. making changes) the existing timestamp should always be overwritten.